xhr.bs

```diff
@@ -2031,6 +2039,19 @@ attributes initialized to false, so it is suggested that for consistency all
 {{ProgressEvent}} interface do the same.
 
 
+<h3 id=feature-policy>Feature Policy Integration</h3>
+
+<p>This specification defines a policy-controlled feature named <dfn>Synchronous
+XMLHttpRequest</dfn>.
+
+<p>The feature name for <a>Synchronous XMLHttpRequest</a> is "sync-xhr".
```

Comment:
xref "feature name"

And make it `"<code>sync-xhr</code>"`.

Reply:
Done. (I've added explicit anchors for the unexported dfns until I fix up the FP spec to export those terms properly)